src/overlays/actors/ovl_En_In/z_en_in.c

```diff
@@ -312,9 +313,10 @@ void func_808F374C(EnIn* this, GlobalContext* globalCtx) {
         Actor_PlaySfxAtPos(&this->actor, NA_SE_VO_IN_CRY_0);
     }
     if (SkelAnime_Update(&this->skelAnime)) {
-        this->unk486 = this->unk488 %= 8;
-        Animation_Change(&this->skelAnime, animations[this->unk488], 1.0f, 0.0f,
-                         Animation_GetLastFrame(animations[this->unk488]), 2, -10.0f);
+        this->unk488 %= 8;
```

Comment:
I guess this makes sense?
```suggestion
        this->unk488 %= ARRAY_COUNT(sAnimations);
```